packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -2131,29 +1918,15 @@ describe('ReactDOMComponent', () => {
         }
       }
 
-      ReactTestUtils.renderIntoDocument(<Parent />, container);
-
-      if (__DEV__) {
-        expect(console.error.calls.count()).toBe(2);
-
-        expect(console.error.calls.argsFor(0)[0]).toContain('className');
-        let matches = console.error.calls
-          .argsFor(0)[0]
-          .match(/.*\(.*:(\d+)\).*/);
-        const previousLine = (matches || [])[1];
-
-        expect(console.error.calls.argsFor(1)[0]).toContain('onClick');
-        matches = console.error.calls.argsFor(1)[0].match(/.*\(.*:(\d+)\).*/);
-        const currentLine = (matches || [])[1];
-
-        //verify line number has a proper relative difference,
-        //since hard coding the line number would make test too brittle
-        expect(parseInt(previousLine, 10) + 12).toBe(parseInt(currentLine, 10));
```

Comment:
I think this was intentional although I don’t remember what exactly it’s trying to verify. Maybe it’s not that important.

Reply:
I read the intent here as making sure that we warned about both issues with distinct component stacks (pointing to the right element for `class` and `onclick`). I'm not _positive_ but that seemed to be the case. I approximated this by changing the element type and asserting with that instead.

Reply:
It looks like several of these lines were added by you a year ago 😅  with PR #8570

The others were added 2 years ago with PR #6398 in order to verify that our error message pointed to the correct component+line. (See issue #6062) I think the new approach I'm taking also satisfies this goal! So it should be okay. 👍